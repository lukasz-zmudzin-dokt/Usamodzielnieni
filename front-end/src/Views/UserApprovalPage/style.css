.background {
    background-image: url("../../assets/menu-backdrop.png");
    background-size: contain;
    background-repeat: no-repeat;
    background-position: center;
    position: relative;

    height: 100vh;
    flex-direction: column;
}

.users-approval-card {
    padding-top: 10vh;
}
.no-border {
    border: none !important;
}
.border {
    border: gray;
}