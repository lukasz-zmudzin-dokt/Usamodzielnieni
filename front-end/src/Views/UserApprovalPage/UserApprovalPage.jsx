import React from "react";
import "./style.css";
import {Card, Container} from "react-bootstrap";
import UsersToApprove from "./components/UsersToApprove";
import NoUsers from "./components/NoUsers";

class UserApprovalPage extends React.Component {
    constructor(props) {
        super(props);
        this.state = { users: [] };
    }

    render() {
        return(
            <Container>
                <div className="background">
                    <Card className="users-approval-card no-border">
                        <Card.Header className="border pb-4"><h3>UÅ¼ytkownicy do akceptacji</h3></Card.Header>
                        <Card.Body className="border">
                            <NoUsers users={this.state.users} />
                            <UsersToApprove users={this.state.users} />
                        </Card.Body>
                    </Card>
                </div>
            </Container>
        );
    }
}

export default UserApprovalPage;