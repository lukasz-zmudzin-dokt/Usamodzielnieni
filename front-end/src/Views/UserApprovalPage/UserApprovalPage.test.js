import React from "react";
import { render } from "@testing-library/react";
import UserApprovalPage from "./UserApprovalPage";

describe("UserApproval", () => {
    it("should render without crashing", () => {
        render(<UserApprovalPage />);
    });

});