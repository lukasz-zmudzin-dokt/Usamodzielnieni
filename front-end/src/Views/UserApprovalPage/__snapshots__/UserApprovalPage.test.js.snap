// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserApproval should look like snapshot 1`] = `
<div
  className="container"
>
  <div
    className="background"
  >
    <div
      className="users-approval-card no-border card"
    >
      <div
        className="border pb-4 card-header"
      >
        <h3>
          Użytkownicy do akceptacji
        </h3>
      </div>
      <div
        className="border card-body"
      >
        <div
          className="card-deck"
        >
          <div
            className="pb-3 col-xl-3 col-lg-4 col-md-6 col-sm-12"
          >
            <div
              className="h-100 card"
            >
              <div
                className="card-body"
              >
                <div
                  className="card-title h5"
                >
                  Maciej
                   
                  Kozłowski
                </div>
                <div
                  className="mb-2 text-muted card-subtitle h6"
                >
                  maciekolejarz64
                </div>
                <div
                  className="list-group list-group-flush"
                  onKeyDown={[Function]}
                >
                  <div
                    className="pl-0 pr-0 list-group-item"
                    data-rb-event-key={null}
                    disabled={false}
                    onClick={[Function]}
                  >
                    <a
                      href="mailto: maciej.kozlowski@gmail.dom"
                    >
                      maciej.kozlowski@gmail.dom
                    </a>
                  </div>
                  <div
                    className="pl-0 pr-0 list-group-item"
                    data-rb-event-key={null}
                    disabled={false}
                    onClick={[Function]}
                  >
                    <p
                      className="mb-1"
                    >
                      NSP 45 im. Adama Małysza
                    </p>
                    <p
                      className="mb-1"
                    >
                      Skocznia narciarska w Zakopanem
                    </p>
                  </div>
                </div>
              </div>
              <div
                className="card-footer"
              >
                <button
                  className="btn btn-success"
                  disabled={false}
                  onClick={[Function]}
                  type="button"
                >
                  Akceptuj
                </button>
                <button
                  className="ml-1 btn btn-danger"
                  disabled={false}
                  onClick={[Function]}
                  type="button"
                >
                  Odrzuć
                </button>
              </div>
            </div>
          </div>
          <div
            className="pb-3 col-xl-3 col-lg-4 col-md-6 col-sm-12"
          >
            <div
              className="h-100 card"
            >
              <div
                className="card-body"
              >
                <div
                  className="card-title h5"
                >
                  Lalka
                   
                  Lalawska
                </div>
                <div
                  className="mb-2 text-muted card-subtitle h6"
                >
                  lalala
                </div>
                <div
                  className="list-group list-group-flush"
                  onKeyDown={[Function]}
                >
                  <div
                    className="pl-0 pr-0 list-group-item"
                    data-rb-event-key={null}
                    disabled={false}
                    onClick={[Function]}
                  >
                    <a
                      href="mailto: lalalalala@gmail.dom"
                    >
                      lalalalala@gmail.dom
                    </a>
                  </div>
                  <div
                    className="pl-0 pr-0 list-group-item"
                    data-rb-event-key={null}
                    disabled={false}
                    onClick={[Function]}
                  >
                    <p
                      className="mb-1"
                    >
                      NSP 45 im. Adama Małysza
                    </p>
                    <p
                      className="mb-1"
                    >
                      Skocznia narciarska w Zakopanem
                    </p>
                  </div>
                </div>
              </div>
              <div
                className="card-footer"
              >
                <button
                  className="btn btn-success"
                  disabled={false}
                  onClick={[Function]}
                  type="button"
                >
                  Akceptuj
                </button>
                <button
                  className="ml-1 btn btn-danger"
                  disabled={false}
                  onClick={[Function]}
                  type="button"
                >
                  Odrzuć
                </button>
              </div>
            </div>
          </div>
          <div
            className="pb-3 col-xl-3 col-lg-4 col-md-6 col-sm-12"
          >
            <div
              className="h-100 card"
            >
              <div
                className="card-body"
              >
                <div
                  className="card-title h5"
                >
                  Ola
                   
                  Maj
                </div>
                <div
                  className="mb-2 text-muted card-subtitle h6"
                >
                  amka
                </div>
                <div
                  className="list-group list-group-flush"
                  onKeyDown={[Function]}
                >
                  <div
                    className="pl-0 pr-0 list-group-item"
                    data-rb-event-key={null}
                    disabled={false}
                    onClick={[Function]}
                  >
                    <a
                      href="mailto: e@ma.il"
                    >
                      e@ma.il
                    </a>
                  </div>
                  <div
                    className="pl-0 pr-0 list-group-item"
                    data-rb-event-key={null}
                    disabled={false}
                    onClick={[Function]}
                  >
                    <p
                      className="mb-1"
                    >
                      NSP 45 im. Adama Małysza
                    </p>
                    <p
                      className="mb-1"
                    >
                      Długa 8
                    </p>
                  </div>
                </div>
              </div>
              <div
                className="card-footer"
              >
                <button
                  className="btn btn-success"
                  disabled={false}
                  onClick={[Function]}
                  type="button"
                >
                  Akceptuj
                </button>
                <button
                  className="ml-1 btn btn-danger"
                  disabled={false}
                  onClick={[Function]}
                  type="button"
                >
                  Odrzuć
                </button>
              </div>
            </div>
          </div>
          <div
            className="pb-3 col-xl-3 col-lg-4 col-md-6 col-sm-12"
          >
            <div
              className="h-100 card"
            >
              <div
                className="card-body"
              >
                <div
                  className="card-title h5"
                >
                  Pablorodriguessenioritazorro
                   
                  Pablorodriguessenioritazorro
                </div>
                <div
                  className="mb-2 text-muted card-subtitle h6"
                >
                  Usernameteżjakiśtakiprzydługi
                </div>
                <div
                  className="list-group list-group-flush"
                  onKeyDown={[Function]}
                >
                  <div
                    className="pl-0 pr-0 list-group-item"
                    data-rb-event-key={null}
                    disabled={false}
                    onClick={[Function]}
                  >
                    <a
                      href="mailto: jakistakidluzszyadresemailowy@email.rerere"
                    >
                      jakistakidluzszyadresemailowy@email.rerere
                    </a>
                  </div>
                  <div
                    className="pl-0 pr-0 list-group-item"
                    data-rb-event-key={null}
                    disabled={false}
                    onClick={[Function]}
                  >
                    <p
                      className="mb-1"
                    >
                      Zespół szkoł wliczających w siebie żłobek, przedszkole i połowę budynków miasta ponieważ jest to jedno wielkie miasteczko uczelniano studencko uczniowskie
                    </p>
                    <p
                      className="mb-1"
                    >
                      ul. Nieuwierzyszżenazwaulicymożebyćtakadługaponieważzdarzasiętoniezwyklerzadko 18, 12-345 Berlin Zachodni
                    </p>
                  </div>
                </div>
              </div>
              <div
                className="card-footer"
              >
                <button
                  className="btn btn-success"
                  disabled={false}
                  onClick={[Function]}
                  type="button"
                >
                  Akceptuj
                </button>
                <button
                  className="ml-1 btn btn-danger"
                  disabled={false}
                  onClick={[Function]}
                  type="button"
                >
                  Odrzuć
                </button>
              </div>
            </div>
          </div>
          <div
            className="pb-3 col-xl-3 col-lg-4 col-md-6 col-sm-12"
          >
            <div
              className="h-100 card"
            >
              <div
                className="card-body"
              >
                <div
                  className="card-title h5"
                >
                  Maciej
                   
                  Kozłowski
                </div>
                <div
                  className="mb-2 text-muted card-subtitle h6"
                >
                  maciekolejarz64
                </div>
                <div
                  className="list-group list-group-flush"
                  onKeyDown={[Function]}
                >
                  <div
                    className="pl-0 pr-0 list-group-item"
                    data-rb-event-key={null}
                    disabled={false}
                    onClick={[Function]}
                  >
                    <a
                      href="mailto: maciej.kozlowski@gmail.dom"
                    >
                      maciej.kozlowski@gmail.dom
                    </a>
                  </div>
                  <div
                    className="pl-0 pr-0 list-group-item"
                    data-rb-event-key={null}
                    disabled={false}
                    onClick={[Function]}
                  >
                    <p
                      className="mb-1"
                    >
                      NSP 45 im. Adama Małysza
                    </p>
                    <p
                      className="mb-1"
                    >
                      Skocznia narciarska w Zakopanem
                    </p>
                  </div>
                </div>
              </div>
              <div
                className="card-footer"
              >
                <button
                  className="btn btn-success"
                  disabled={false}
                  onClick={[Function]}
                  type="button"
                >
                  Akceptuj
                </button>
                <button
                  className="ml-1 btn btn-danger"
                  disabled={false}
                  onClick={[Function]}
                  type="button"
                >
                  Odrzuć
                </button>
              </div>
            </div>
          </div>
          <div
            className="pb-3 col-xl-3 col-lg-4 col-md-6 col-sm-12"
          >
            <div
              className="h-100 card"
            >
              <div
                className="card-body"
              >
                <div
                  className="card-title h5"
                >
                  Maciej
                   
                  Kozłowski
                </div>
                <div
                  className="mb-2 text-muted card-subtitle h6"
                >
                  maciekolejarz64
                </div>
                <div
                  className="list-group list-group-flush"
                  onKeyDown={[Function]}
                >
                  <div
                    className="pl-0 pr-0 list-group-item"
                    data-rb-event-key={null}
                    disabled={false}
                    onClick={[Function]}
                  >
                    <a
                      href="mailto: maciej.kozlowski@gmail.dom"
                    >
                      maciej.kozlowski@gmail.dom
                    </a>
                  </div>
                  <div
                    className="pl-0 pr-0 list-group-item"
                    data-rb-event-key={null}
                    disabled={false}
                    onClick={[Function]}
                  >
                    <p
                      className="mb-1"
                    >
                      NSP 45 im. Adama Małysza
                    </p>
                    <p
                      className="mb-1"
                    >
                      Skocznia narciarska w Zakopanem
                    </p>
                  </div>
                </div>
              </div>
              <div
                className="card-footer"
              >
                <button
                  className="btn btn-success"
                  disabled={false}
                  onClick={[Function]}
                  type="button"
                >
                  Akceptuj
                </button>
                <button
                  className="ml-1 btn btn-danger"
                  disabled={false}
                  onClick={[Function]}
                  type="button"
                >
                  Odrzuć
                </button>
              </div>
            </div>
          </div>
          <div
            className="pb-3 col-xl-3 col-lg-4 col-md-6 col-sm-12"
          >
            <div
              className="h-100 card"
            >
              <div
                className="card-body"
              >
                <div
                  className="card-title h5"
                >
                  Maciej
                   
                  Kozłowski
                </div>
                <div
                  className="mb-2 text-muted card-subtitle h6"
                >
                  maciekolejarz64
                </div>
                <div
                  className="list-group list-group-flush"
                  onKeyDown={[Function]}
                >
                  <div
                    className="pl-0 pr-0 list-group-item"
                    data-rb-event-key={null}
                    disabled={false}
                    onClick={[Function]}
                  >
                    <a
                      href="mailto: maciej.kozlowski@gmail.dom"
                    >
                      maciej.kozlowski@gmail.dom
                    </a>
                  </div>
                  <div
                    className="pl-0 pr-0 list-group-item"
                    data-rb-event-key={null}
                    disabled={false}
                    onClick={[Function]}
                  >
                    <p
                      className="mb-1"
                    >
                      NSP 45 im. Adama Małysza
                    </p>
                    <p
                      className="mb-1"
                    >
                      Skocznia narciarska w Zakopanem
                    </p>
                  </div>
                </div>
              </div>
              <div
                className="card-footer"
              >
                <button
                  className="btn btn-success"
                  disabled={false}
                  onClick={[Function]}
                  type="button"
                >
                  Akceptuj
                </button>
                <button
                  className="ml-1 btn btn-danger"
                  disabled={false}
                  onClick={[Function]}
                  type="button"
                >
                  Odrzuć
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
